pkg/tsdb/influxdb/query_part.go

```diff
@@ -50,7 +59,7 @@ func init() {
 	renders["stddev"] = QueryDefinition{Renderer: functionRenderer}
 	renders["time"] = QueryDefinition{
 		Renderer: functionRenderer,
-		Params:   []DefinitionParameters{{Name: "interval", Type: "time"}},
+		Params:   []DefinitionParameters{{Name: "interval", Type: "time"}, {Name: "offset", Type: "time"}},
```

Comment:
Afraid this is a breaking change? What of people running on older versions of inflxudb where the offset parameter does not exist?